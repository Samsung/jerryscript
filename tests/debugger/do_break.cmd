break do_break.js:24
b do_break.js:27
b do_break.js:17
list
delete 2
next
c
