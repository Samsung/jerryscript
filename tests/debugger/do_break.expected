Connecting to: localhost:5001
Stopped at tests/debugger/do_break.js:15
(jerry-debugger) Breakpoint 1 at tests/debugger/do_break.js:24 (in test)
(jerry-debugger) Breakpoint 2 at tests/debugger/do_break.js:27 (in test)
(jerry-debugger) Breakpoint 3 at tests/debugger/do_break.js:17
(jerry-debugger) 1: tests/debugger/do_break.js:24 (in test)
2: tests/debugger/do_break.js:27 (in test)
3: tests/debugger/do_break.js:17
(jerry-debugger) (jerry-debugger) Stopped at breakpoint:3 tests/debugger/do_break.js:17
(jerry-debugger) Press enter to stop JavaScript execution.
Connection closed.
